ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1090,8 +1090,7 @@ private enum UnityLayers {
     // Advance the lagging vessels and kill those which collided with a
     // celestial.
     {
-      DisposableIterator collided_vessels;
-      plugin_.CatchUpLaggingVessels(out collided_vessels);
+      plugin_.CatchUpLaggingVessels(out DisposableIterator collided_vessels);
```

Comment:
This is a perfectly valid cleanup, but I'd rather avoid this stuff with mixing semantics as it makes the review really hard.  The same holds for the places where string concatenations are replaced by `$@""` strings.

Could you move all the cleanups to a different PR that would go in first?

Reply:
Done in #2172.